docs/contributing/core/index.md

```diff
@@ -73,86 +71,58 @@ On macOS, Developer Tools provides Clang for a C compiler.
 On Linux, [GNU Compiler Collection (GCC)](https://gcc.gnu.org/) is a common option.
 
 
-
+(contributing-core)=
 ## Install Plone core for development
 
 The tool that installs Plone core is `buildout.coredev`.
 
-The current default and development branch of `buildout.coredev` is `6.1`
+The current default and development branch of `buildout.coredev` is `{PLONE_BACKEND_MINOR_VERSION}`.
 Older versions are named according to their `major.minor` version.
 Its versions align with Plone's `major.minor` versions.
 
 Use a separate directory for each version of Plone to which you want to contribute.
-This will avoid switching between git branches, then re-running buildout, which can cause dependency conflicts between versions of Plone.
+This will avoid switching between Git branches, which can cause dependency conflicts between versions of Plone.
 
-To set up a Plone 6 development environment, change your working directory to wherever you place your projects, and clone https://github.com/plone/buildout.coredev.
-You can specify the branch that you want to check out with the `-b` option.
+To set up a Plone 6 development environment, clone https://github.com/plone/buildout.coredev.
 
 ```shell
-cd [MY_PROJECTS]
-# clone a specific major.minor version branch
-git clone -b 6.1 https://github.com/plone/buildout.coredev
+git clone https://github.com/plone/buildout.coredev
 cd buildout.coredev
 ```
 
-````{important}
-If you want to use a Python version that is not 3.11, follow these instructions.
-
-Open the file {file}`bootstrap.sh` at the root of the repository.
-Notice that the script expects Python 3.11 to be installed on your system and in your user's `PATH`.
-
-```shell
-#/bin/sh
-`which python3.11` -m venv .
-```
-
-Edit it according to the Python version you want to use, then save and close the file.
-After you have run the script, you should undo the change, otherwise you have a local change in git that you might accidentally commit.
-````
-
-Now run the script to install Plone 6.
-
-```shell
-./bootstrap.sh
-```
-
-This will run for a long time if it's your first pull (approximately 10-20 minutes, depending on network speed and your hardware).
-
-Once that's done, you can start an instance of Plone with the following command.
+Use the following command to install Plone and run it.
 
 ```shell
-./bin/instance fg
+make run
 ```
 
-```{include} /_inc/_create-classic-ui-instance.md
-```
+To create and visit a new Plone site, visit http://localhost:8080, select a distribution, and fill out and submit the form, ensuring you create a site with sample content against which you can test.
```

Comment:
Reverting to previous version since this page is intended exclusively for contributing to the Plone core backend.

```suggestion
To create and visit a new Plone site, visit http://localhost:8080, select the Classic UI distribution, and fill out and submit the form, ensuring you create a site with sample content against which you can test.
```